internal/cmd/controller/gitops/reconciler/status_controller.go

```diff
@@ -161,7 +161,6 @@ func (r *StatusReconciler) updateStatus(ctx context.Context, orig *fleet.GitRepo
 }
 
 func setStatus(list *fleet.BundleDeploymentList, gitrepo *fleet.GitRepo) error {
-	// sort for resourceKey?
```

Comment:
I think these comments actually meant `namespace/name` rather than _those_ `ResourceKey`s
(e.g. if they would make it a more comprehensive list for users)

Reply:
Replacing with a comment that explains why we sort.